crates/sui-storage/src/event_store/sql.rs

```diff
@@ -63,8 +65,25 @@ enum EventsTableColumns {
     ObjectId,
     /// fields TEXT
     Fields,
+    /// type_struct TEXT
+    TypeStruct,
```

Comment:
This is really the move event type name.  Can we please call this column `MoveType` or `MoveEvent`?   It's definitely not a struct...  just for clarity it would help

Reply:
Or maybe `EventName`.  It is the full MOVE event name....

Reply:
it's called `StructTag` (https://github.com/diem/move/blob/main/language/move-core/types/src/language_storage.rs#L40) , I misspelled it. will correct

Reply:
Right, my point is that you aren't storing the original tag.  You are just storing the name of it.  A struct would have members, field names and types which this doesn't have.

Reply:
will update